sphinx_gallery/gen_rst.py

```diff
@@ -124,7 +125,7 @@ def flush(self):
 """
 
 
-CODE_OUTPUT = """.. rst-class:: sphx-glr-script-out
+CODE_OUTPUT = u""".. rst-class:: sphx-glr-script-out
```

Comment:
unicode_literals takes the need away to prefix every string with `u`, makes everything unicode. My first guess was to use that instead of going through each string prefixing the `u`. But it is not that perfect, and I could not get sphinx-gallery to run my examples with unicode_literals in gen_rst.py